internal/github/lastRepo.go

```diff
@@ -0,0 +1,66 @@
+package github
```

Comment:
maybe we can call this filename `repositories.go` and it can hold more repository level stuff
this would also align with some of our other file names in the github package

Reply:
Yes I agree. Thank you for taking the time to examine this code and give good feedback. I will make your suggested changes. After I commit and push these changes do I submit another Pull Request?

Reply:
nope, you just need to add commits and it will update this PR!

Reply:
well make the commits locally and just `git push`. this PR will reflect those commits!

Reply:
Thank you!